providers/base/units/graphics/jobs.pxu

```diff
@@ -528,6 +533,7 @@ category_id: com.canonical.plainbox::graphics
 id: graphics/{index}_auto_switch_card_{product_slug}
 requires:
  graphics_card.driver in ['nvidia', 'amdgpu-pro', 'pcieport']
+ graphics_card.driver != 'amdgpu-pro' and lsb.release < '22.04'
```

Comment:
I'm always weary of multiple requirements (even though [it should work](https://checkbox.readthedocs.io/en/latest/units/job.html)), because it can lead to weird stuff that is hard to debug.

Did you try on a 20.04 machine to make sure it worked as expected? (the submissions you listed are all 22.04)